src/panel.js

```diff
@@ -116,6 +149,37 @@ const addLearnHowFBCWorksButton = async (fragment) => {
   addSpan(button, "sites-added-subhead");
 };
 
+const addCustomSiteButton = async (fragment, panelId) => {
+  const shouldShowRemoveSiteButton = await isSiteInContainer(panelId);
+  let button;
+  if (shouldShowRemoveSiteButton) {
+    button = addFullWidthButton(fragment, "remove-site-from-container");
+    addSpan(button, "button-remove-site");
+    addTooltip(button, "button-remove-site-tooltip");
+  } else {
+    button = addFullWidthButton(fragment, "add-site-to-container");
+    addSpan(button, "button-allow-site");
+  }
+};
+
+
+const setCustomSiteButtonEvent = async (panelId) => {
+  const shouldShowRemoveSiteButton = await isSiteInContainer(panelId);
+
+  if (panelId === "on-facebook") {
+    const removeSiteFromContainerLink = document.querySelector(".remove-site-from-container");
+    removeSiteFromContainerLink.classList.add("disabled-button");
+    return;
+  }
+
+  if (shouldShowRemoveSiteButton) {
+    const removeSiteFromContainerLink = document.querySelector(".remove-site-from-container");
+    removeSiteFromContainerLink.addEventListener("click", async () => removeSiteFromContainer());
```

Comment:
nit: can remove the `else` block here with a `return;` early statement, like in the above `if (panelId === "on-facebook") {` block.